core/src/components/side-menu/readme.md

```diff
@@ -1,6 +1,21 @@
 # tds-side-menu
 
-
+### Side Menu is a component that consists of these sub-components:
+
+ - side-menu-user-label
+ - side-menu-user-image
+ - side-menu-user
+ - side-menu-overlay
+ - side-menu-item
+ - side-menu-dropdown
+ - side-menu-dropdown-list
+ - side-menu-dropdown-list-item
+ - side-menu-collapse-button
+ - side-menu-close-button
+
+Some of the upper mentioned subcomponents are not listed under <i>Notes</i> tab
+because either they are used only internally or they have no props or events but serve as wrappers.
```

Comment:
I think it's nice if subcomponents that are only wrappers still are listed under Notes, so it's clear that they exist and what props they have, even if they don't have any.

Also, if a component is only used internally I think it's better to not mention it in documentation at all, so it remains an implementation detail that we can change and/or remove easily and without updating the docs.

Reply:
Good comment, I will double check then which one we use as wrappers as expose then and keep hidden ones that are used internally ;)

Reply:
PR updated! ;)